docs/assets/templates.md

```diff
@@ -0,0 +1,88 @@
+# Templates Overview
+
+We simplify the creation of pipeline projects through the use of **templates**. Templates provide a starting point with predefined structures, configurations, and connections to help you quickly set up and standardize your pipelines.
+
+---
+
+## What Are Templates?
+
+Templates in Bruin are blueprints for initializing new pipeline projects. They:
+- Define the structure and essential files for a pipeline.
+- Contain boilerplate code and configurations tailored to specific use cases.
+- Allow users to quickly set up pipelines without starting from scratch.
+
+By using templates, developers can focus on their pipeline logic rather than worrying about setup and configuration.
+
+---
+
+## How to Use Templates
+
+Using templates in Bruin is straightforward with the `bruin init` command:
+
+```bash
+bruin init [template-name] [folder-name]
+```
+Arguments:
+
+- **template-name** (optional):  
+  The name of the template to use. If omitted, the default template is used.
+
+- **folder-name** (optional):  
+  The name of the folder where the project will be created. If omitted, the folder name will be:  
+    - `bruin-pipeline` for the default template.  
+    - The template name for custom templates.
```

Comment:
they can actually just run `bruin init` and it'll show a wizard. I think we should recommend doing that instead, and still explain those options.